lib/src/stomp_config.dart

```diff
@@ -1,4 +1,5 @@
 import 'dart:async';
+import 'dart:io';
```

Comment:
Have you tested this on web? Or ran the unit-tests? I am bit concerned with unconditional import here.

Reply:
> Have you tested this on web?
Yes, it will work on the Web since HTTP client creation is used only on mobile. Import is not a problem and will compile and work well.
>  Or ran the unit-tests? 
Yes all tests expect 2 a passed(they are failing in master as well). Related to StompParser

Reply:
Ah, I run test wrong way.
00:29 +104: All tests passed! 
All tests passed